src/WebJobs.Script.WebHost/Diagnostics/Extensions/ScriptHostServiceLoggerExtension.cs

```diff
@@ -332,7 +332,13 @@ public static void ScriptHostServiceRestartCanceledByRuntime(this ILogger logger
 
         public static void LogHostInitializationSettings(this ILogger logger, string functionWorkerRuntime, string functionExtensionVersion, string currentDirectory, bool inStandbyMode)
         {
-            _logHostInitializationSettings(logger, functionWorkerRuntime, functionExtensionVersion, currentDirectory, inStandbyMode, null);
+            var hostInitializationSettings = $@"Settings: {{
+    ""functionsWorkerRuntime"": ""{functionWorkerRuntime}"",
```

Comment:
Question: I see we are not logging a JSON string in other places, Ex: https://github.com/Azure/azure-functions-host/blob/aae3d338434532ea4998049e4e9ee69a6fe039b7/src/WebJobs.Script/Diagnostics/Extensions/ScriptHostLoggerExtension.cs#L142C59-L142C59

Did we want to log this as a string which contains a JSON string?

Reply:
Are we able to parse that in kusto? I think we decided to log the changes as a JSON string so that it would be easy to parse

Reply:
Talked offline and we decided to format the string to be a proper JSON string:
{ "HostInitializationSettings" : { ..} }